aws-athena/views/qc-vw_class_mismatch.sql

```diff
@@ -0,0 +1,31 @@
+SELECT
+    pardat.taxyr,
+    pardat.parid,
+    legdat.user1 AS township_code,
+    pardat.class AS pardat_class,
+    comdat.class AS comdat_class,
+    dweldat.class AS dweldat_class,
```

Comment:
Only because it was included in the original Inquire query! I moved the `dweldat` class test into the table schema since it's a little more complicated than the ones performed here.